client/src/modules/reports/generate/report_accounts/report_accounts.config.js

```diff
@@ -0,0 +1,75 @@
+angular.module('bhima.controllers')
+  .controller('report_accountsController', ReportAccountsConfigController);
```

Comment:
The naming convention here is vary confusing... The controller is `report_accounts`, but the AppCache is `reports_account` and the route is `report/finance/account`.

I propose that we change all these to `account_report` - no plurals.  That way, we won't have to remember which word the `s` goes on.  All variables should be updated accordingly.